README.md

```diff
@@ -0,0 +1,219 @@
+
+## Expectations
+
+### What you can expect of me
+
+Here is a short list of behaviors you should expect from me at all times. If I'm ever failing to meet any of the expectations outlined in this section, please let me know immediately. This is not an exhaustive list of expectations but aims to cover those that are most vital.
+
+#### You will be a top priority
+
+It's my belief that the people who report to me are _the job_. I'm tasked with various other responsibilities as an Engineering Manager but none are higher priority in my view than ensuring my direct reports are getting what they need to be successful. In practice, this means you should expect that I will make time for you when you need it, follow through on commitments I make to you, and help you deal with whatever challenges you encounter.
+
+#### Give my best in everything I do
+
+This is a foundational principle for how I approach everything in my life. If I'm going to do something, I try to do it as well as I can. This doesn't mean that I will always deliver outstanding results, just that I'll aim to do my best. In practice, this means you should expect me to give you a great effort on every front from our 1:1 meetings to performance evaluations to helping you navigate challenges.
+
+#### Default to transparent
+
+I believe organizations operate at their highest level when information is not restricted. In the interest of this, I commit to being transparent and to proactively share as much as I can. The flip side of this is that you should feel epowered to ask me for information. Two important exceptions to this policy of transparency are sensitive personal information and conversations around performance; I will always keep these confidential.
+
+#### Default to trust
+
+Simply put, I'll trust you until you give me a reason not to. I want my direct reports to have a lot of autonomy and that's only possible when paired with trust. You shouldn't find me micromanaging your work, schedule, or decision-making. I'll likely ask you a lot of probing questions but this is generally due to curiosity or a desire to gain more context.
+
+#### Feedback will be delivered in a timely fashion
+
+Giving feedback, both positive and constructive, is one of the most important functions I perform in my role as a manager. Since feedback has such high value, not giving it to you as soon as I have it would be doing you a disservice. You'll often find me giving you feedback right away, in the moment when I perceive something. When it's not possible to provide that realtime feedback or when I come to have feedback outside of direct interactions with you, I will provide this to you in our next 1:1. It's my goal for you to always know how you're doing. If there are surprises for you in a performance feedback loop, it's an indication that I'm not being effective at this.
```

Comment:
Would be interested in hearing more on how some of this feedback will be fueled. For instance, is it primarily interaction or deadline focused. How can other performance metrics (Bugs closed, Feature launched, PRs reviewed, Paired programmings done) influence more objective metrics.

Reply:
This is going to be a bit different in any job//role I find myself in.

I am realizing now, that we might be having a bit of a misunderstanding of what I want this artifact to be. My intent is to be job-agnostic. That is, this artifact is not tied to the job that I currently find myself employed in. So, in this section and in several others where you've provided feedback, my preference is to highlight the principles I'll adhere to or the high-level concepts that will guide me rather than getting into the implementation details of how this might work in any given job.

Now, this begs the question: have I been consistent with this throughout? I'll review and ensure that I correct if that's not the case.

In any event, to answer your specific question, it's a bit of everything in my current role at Optimizely. I'll see things myself (say, in a 1:1 or a team meeting or a document), I'll keep an eye on the tools I have at my disposal (GH, Jira), and I'll have conversations with people that work with my direct reports in order to gather additional insights I wouldn't otherwise be privy to. However, in a different job, this might look quite different. I don't want to couple this doc to what happens in my job at Optimizely (or anywhere else).